internal/services/cdn/cdn_frontdoor_origin_resource.go

```diff
@@ -179,12 +179,14 @@ func resourceCdnFrontDoorOriginCreate(d *pluginsdk.ResourceData, meta interface{
 	ctx, cancel := timeouts.ForCreate(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	originGroupId, err := parse.FrontDoorOriginGroupID(d.Get("cdn_frontdoor_origin_group_id").(string))
+	originGroupRaw := d.Get("cdn_frontdoor_origin_group_id").(string)
+	originGroup, err := parse.FrontDoorOriginGroupIDInsensitively(originGroupRaw)
```

Comment:
this is coming from the Resource ID and so should be being parsed case-sensitively:

```suggestion
	originGroup, err := parse.FrontDoorOriginGroupID(originGroupRaw)
```

Reply:
Fixed.